build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -21,11 +21,22 @@ const jsModuleErrorsExtension = '.ddc.js.errors';
 const jsModuleExtension = '.ddc.js';
 const jsSourceMapExtension = '.ddc.js.map';
 const metadataExtension = '.ddc.js.metadata';
+const fullKernelExtension = '.ddc.full.dill';
 
 /// A builder which can output ddc modules!
 class DevCompilerBuilder implements Builder {
   final bool useIncrementalCompiler;
 
+  /// Make ddc generate full dill for libraries it compiles.
+  ///
+  /// Full dill is only generated in debug mode and does not include any
+  /// dependent libraries or SDK dill.
+  ///
+  /// Full dill is used by the expression compilation service run by the
+  /// debugger to compile expressions in debugging worklows that use modular
+  ///  build, such as webdev and google3.
```

Comment:
This package isn't used inside Google3 at all so we can probably leave out this extra bit of detail.

Reply:
thanks, fixed